### ISSUE TITLE
`in` - inverse of `contains`

### ISSUE THREAD

**Issue by cvogt** [scala-extensions/scalax#13]
```
  implicit class AnyExtensions[T](val value: T) extends AnyVal{
    /** Tests whether the given sequence contains this value as an element */
    def in(seq: Seq[T]) = seq.contains(value)
    /** Tests whether the given sequence does NOT contains this value as an element */
    def notIn(seq: Seq[T]) = !seq.contains(value)
    /** Tests whether the given set contains this value as an element */
    def in(set: Set[T]) = set.contains(value)
    /** Tests whether the given set does NOT contains this value as an element */
    def notIn(set: Set[T]) = !set.contains(value)
  }
```

https://github.com/cvogt/scala-extensions/blob/master/src/main/scala/scalar.scala

Not sure we want extensions on Any though?

**Comment by t3hnar** [comment]
I'd avoid methods with Seq, as those are not supposed to be fast with `contains`

**Comment by cvogt** [comment]
I'd argue, that with regards to performance and correctness, `in` here is no different to the std lib's contains. So it does qualify as "following the same style", if that's a metric we want to use.

(Apart form the std lib, I think performance does not matter in many cases, where collections are small, but correctness is a problem, because .contains is defined on Any).

**Comment by t3hnar** [comment]
@cvogt so then you should stick to `Traversable` / `Iterable`

**Comment by cvogt** [comment]
Yeah, we can absolutely generalize this to the same level as .contains .

**Comment by jozic** [comment]
have a mixed feeling about this one, it looks nice, but doesn't buy you much, compared to other things we are going to have, like `foldLeftWhile`
i'm totally fine with writing `List(1,2,3) contains 1` instead of `1 in List(1,2,3)`
i'm not strongly opposed it, though
